charts/cadc-tap/values.yaml

```diff
@@ -158,14 +164,7 @@ tapSchema:
     tag: "2.6.1"
 
   # -- Resource limits and requests for the TAP schema database pod
-  # @default -- See `values.yaml`
-  resources:
-    limits:
-      cpu: "1"
-      memory: "1Gi"
-    requests:
-      cpu: "5m"
-      memory: "400Mi"
+  resources: {}
```

Comment:
Please don't remove the resources.  If these are not the right default values then fix them.

Reply:
Didn't mean to remove those, they should be back now thank for noticing